lib/ansible/modules/database/mysql/mysql_user.py

```diff
@@ -435,7 +443,8 @@ def pick(x):
             return x
 
     for grant in grants:
-        res = re.match("""GRANT (.+) ON (.+) TO (['`"]).*\\3@(['`"]).*\\4( IDENTIFIED BY PASSWORD (['`"]).+\5)? ?(.*)""", grant[0])
+        res = re.match(
+            """GRANT (.+) ON (.+) TO (['`"]).*\\3@(['`"]).*\\4( IDENTIFIED BY PASSWORD (['`"]).+\5)? ?(.*)""", grant[0])
```

Comment:
You'll need to rebase to fix the regex (see https://github.com/ansible/ansible/pull/52278/files#diff-399cf46d54197b26de359e13cb8bc033R430 )